app/models/agenda-item.ts

```diff
@@ -49,6 +49,61 @@ export default class AgendaItemModel extends Model {
   get dateFormatted() {
     return this.session?.dateFormatted;
   }
+
+  get agendaItemQuality(): {
+    qualityMetrics: { label: string; value: boolean }[];
+    qualityValue: number;
```

Comment:
@snenenenenenene Is it use somewhere?

Reply:
I've removed the quality value, it was a remnant of what was based on the mock ups. 
But it does seem we won't need it in the near future so I've deleted it!